docs/curation-review-tut.md

```diff
@@ -0,0 +1,417 @@
+ - easy for readers to explore your dataset (with no extra work on your side - goodbye shiny!)
+ - accesibility of your dataset through many of the popular single cell toolchains (link to an instance of the cellxgene explorer app)
+
+<br/>
+ 
+## Basic data requirements
+
+<br/>
+
+The following elements of your single cell dataset are required. One of the prerequisites for contributing data to the portal is that you have these components represented as an `anndata` object (more on that in the next section). Note that, in addition to the following, other representations of the expression matrix (alternative normalization, SCTransform, corrected counts from SCTransform or background correction) can all be stored as `layers` your anndata object.
+
+- count matrix (except for certain assays)
+- normalized expression matrix used for visulization
+- cell level metadata
+- variable/feature level metadata (not required)
+- embedding (at least one required, UMAP, tSNE, spatial, PCA)
+
+
+
+<br/>
+
+
+## Anndata object requirements
+
+<br/>
+
+Contributing data to the cellxgene data portal or visualizing your data with the cellxgene explorer requires that your data is represented as an anndata data object. You can read more about Anndata through its [documentation site](https://anndata.readthedocs.io/en/latest/). Additionally, you can check out some of the requirements for using an anndata object with cellxgene [here](https://chanzuckerberg.github.io/cellxgene/posts/prepare). In this section of the tutorial, we will augment some of this information with some considerations that may be specific to the cellxgene curation process.
+
+<br/>
+
+<details>
+<summary>Anndata description</summary>
+
+<p align="center">
+  <img width="500" src="https://user-images.githubusercontent.com/25663501/111377611-3c8c7400-8677-11eb-8176-cf9de8f64c70.png">
+</p>
+
+<p align="center">
+  <b> Figure:</b> Anndata components
+</p>
+
+This is some text introducting anndata
+
+
+- Talk about object structure
+- limitations 
+- what are the features/aspects of the anndata object that cellxgene data portal expects (essential slots/entries in the anndata object) - talk about where schema information is stored (schema info stored in obs and uns)
+- obs
+- uns
+- obsm ("X_example")
+- X
+- raw.X or raw layer
+- layers
+
+</details>
+
+<br/>
+ 
+<details>
+<summary>Format conversion (i.e. Seurat => Anndata) up</summary>
+ 
+Talk about conversion - sceasy, other tools....
+ 
+</details>
+
+<br/>
+ 
+<details>
+<summary>Alternative assays</summary>
+
+Talk about the data types that we are able to support (the tool is mostly ambiguous, but anndata does have some limitations that can be mentioned about here)
+
+Guidelines for curating the following assays. In each section, also include a link to curation of an object of the specified assay type:
+- ATAC/mC
+- RNAseq
+- CITEseq
+- Spatial/Visium
+
+</details>
+
+<br/>
+
+## Schema definition
```

Comment:
Assuming that schema definition will be updated to reflect the requirements discussion.

Reply:
I think that this should link to the schema reference. We don't want to keep duplicates updated when versioning. We've already ran into problems with multiple documents related to schema.